### PR TITLE
[#173487093] Add support & signup forms

### PR DESCRIPTION
**Pull request by kr8n3r**
Massive thanks to @paroxp for his support and contribution to this PR.

What
----

This is an implementation of current support forms found on product pages
- https://www.cloud.service.gov.uk/support
- https://www.cloud.service.gov.uk/signup
- https://www.cloud.service.gov.uk/contact-us

into a public Paas admin /support route.

For `get/post` routes i opted to use different handler, so as to make it more readable for myself instead of having a single complex `/support/:route`  handler.

Content of forms, confirmation pages and emails follows what is currently there.

Why
----

We are building new product pages which are fully static HTML. Additionally, bringing support form into here should allow as to evolve these forms in the future with app and user data to make them even more effortless. 

How to review
-------------
- checkout `support-page` branch
- run `npm install`
- Run locally with `ZENDESK_API_TOKEN="{token}" ZENDESK_USERNAME="{user}" npm run start:with-stub-api` with creds from the password-store (or deploy to your env)
- check forms and error message
- evaluate code

Note: tried to split work into logical commits. later on changes applied across features so best to look at the combination of commit-by-commit and whole end result file

Who can review
---------------
not @kr8n3r 

Visuals
---------------

<details>
<summary>Visual changes</summary>
Header breakpoints have been adjusted to allow for more navigation elements<br/><br/>

paas-header
</details>

<details>
<summary>Example form with error messages</summary>

0 0 0 0_3000_support_something-wrong-with-service
</details>

<details>
<summary>Example confirmation page</summary>
Confirmation page content is different for each type of support request<br/><br/>

0 0 0 0_3000_support_something-wrong-with-service (1)
</details>

<details>
<summary>Zendesk emails</summary>
Example of Zendesk email content and subject line. <br/>

Screenshot 2020-07-16 at 07 25 47 <br/>

Screenshot 2020-07-16 at 07 26 56<br/>

Screenshot 2020-07-16 at 07 27 42<br/>
</details>

### CODE REVIEW

**Comment by kr8n3r** [comment]
requires https://github.com/alphagov/paas-cf/pull/2379 before merging

File: src/components/support/views.tsx [hunk 1]
```diff
@@ -162,4 +177,302 @@ export function SupportSelectionPage(props: ISupportSelectionFormProperties): Re
...
+            Submit
+          </button>
+        </form>
+        <h2 className="govuk-heading-m">Escalate a request</h2>
+        <p className="govuk-body">If you have already sent your request for support and you think we’re not handling it in the way you would expect, please contact a member of the product team, who will reply during working hours.</p>
+        <p className="govuk-body">To escalate an issue about a production service in or outside working hours, please use the emergency contact details you have been sent.</p>
+      </div>
+    </div>
+  );
 }
```
Comment (LeePorte) [inline src/components/support/views.tsx]: newline please

File: src/components/support/controllers.tsx [hunk 2]
```diff
@@ -210,3 +221,56 @@ export async function HandleSomethingWrongWithServiceFormPost (ctx: IContext, pa
...
+          Read more about our{' '}
+          <a className="govuk-link" 
+            href="https://www.cloud.service.gov.uk/support-and-response-times">
+              support and resolution times
+          </a>.
+        </SupportConfirmationPage>,
+      ),
+    };
+  }
+}
```
Comment (LeePorte) [inline src/components/support/controllers.tsx]: newline please-

File: src/components/support/views.tsx (hunk 1 above)
Comment (kr8n3r) [reply src/components/support/views.tsx]: resolved in later commits

File: src/components/support/controllers.tsx (hunk 2 above)
Comment (kr8n3r) [reply src/components/support/controllers.tsx]: resolved in later commit

**Pull request merged**